pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -140,61 +141,97 @@ func createDnsmasqConfiglet(
 	dhcphostsDir := dnsmasqDhcpHostDir(bridgeName)
 	ensureDir(dhcphostsDir)
 
-	file.WriteString(dnsmasqStatic)
+	var b bytes.Buffer
+
+	createDnsmasqConfigletNoFile(&b, ctx, bridgeName, bridgeIPAddr, netstatus, hostsDir, ipsetHosts, uplink, dnsServers, ntpServers)
+
+	file.WriteString(b.String())
+}
+
+func dhcpRangeConfigLine(start, end net.IP) (string, error) {
+	var dhcpRange string
+
+	if start != nil {
+		if end == nil {
+			dhcpRange = fmt.Sprintf("%s,static", start.String())
+		} else {
+			if bytes.Compare(start, end) > 0 {
+				return dhcpRange, fmt.Errorf("createDnsmasqConfigletWithBuffer: invalid dhcp-range %s-%s",
+					start.String(), end.String())
+			}
+			// full dhcp-range is not possible with static, see https://thekelleys.org.uk/dnsmasq/docs/dnsmasq-man.html
+			// "dhcp-range=[tag:<tag>[,tag:<tag>],][set:<tag>,]<start-addr>[,<end-addr>|<mode>[,<netmask>[,<broadcast>]]][,<lease time>]"
+			dhcpRange = fmt.Sprintf("%s,%s", start.String(), end.String())
```

Comment:
Since it is no longer explicitly configured, but instead relies on the default value, could you please also add a comment that the lease time for IPv4 address is 60 minutes?

Reply:
it is still setting the 60m, see https://github.com/lf-edge/eve/pull/3028/files#diff-7f0e571b7706f2026efaa28af4c13ea050fa9b52cf6b5b2a0a3f2ee2bbd6291aR336 but perhaps the method name is misleading ...
perhaps `dhcpv4RangeConfig`

Reply:
Ah, OK, I missed that part.